src/main/java/com/kodilla/ecommercee/mapper/UserMapper.java

```diff
@@ -18,6 +18,7 @@ public User mapToUser(final UserDto userDto) {
 
     public UserDto mapToUserDto(final User user) {
         return new UserDto(
+                user.getId(),
                 user.getUsername(),
                 user.getStatus(),
                 user.getUserKey());
```

Comment:
Zauważyłem że metoda `mapUserDtoList` nie jest nigdzie wykorzystywana to można się jej pozbyć.